tests/testthat/testdata/v4.0.1-tasks-fail-round-id-pattern.json

```diff
@@ -0,0 +1,277 @@
+{
+    "schema_version": "https://raw.githubusercontent.com/hubverse-org/schemas/main/v4.0.1/tasks-schema.json",
+    "rounds": [{
+            "round_id_from_variable": true,
+            "round_id": "round_id_var",
+            "model_tasks": [{
+                "task_ids": {
+                    "round_id_var": {
+                        "required": null,
+                        "optional": ["24_25_covid", "invalid-round-id-in-var"]
```

Comment:
These invalida values are not flagged yet because this config fails basic validation against the schema. Hence dynamic validations are not run